packages/wallet/src/redux/protocols/consensus-update/__tests__/scenarios.ts

```diff
@@ -36,92 +53,267 @@ const ledger7 = ledgerCommitment({
   balances: twoThreeOneTwo,
 });
 
+const threePlayerLedger6 = threeWayLedgerCommitment({ turnNum: 6, balances: twoThreeOne });
+const threePlayerLedger7 = threeWayLedgerCommitment({ turnNum: 7, balances: twoThreeOne });
+const threePlayerLedger8 = threeWayLedgerCommitment({ turnNum: 8, balances: twoThreeOne });
+const threePlayerLedger9 = threeWayLedgerCommitment({
+  turnNum: 9,
+  balances: twoThreeOne,
+  proposedBalances: oneOneFour,
+});
+const threePlayerLedger10 = threeWayLedgerCommitment({
+  turnNum: 10,
+  balances: twoThreeOne,
+  proposedBalances: oneOneFour,
+  isVote: true,
+});
+const threePlayerLedger11 = threeWayLedgerCommitment({
+  turnNum: 11,
+  balances: oneOneFour,
+});
+
 // ------
 // SharedData
 // ------
-const aInitialSharedData = setChannels(EMPTY_SHARED_DATA, [
+
+const threePlayerInitialSharedData = (ourIndex: ThreePartyPlayerIndex) => {
+  const address =
+    ourIndex === ThreePartyPlayerIndex.A
```

Comment:
Why not have a lookup table
```
{
  [ThreePartyPlayerIndex.A]: asAddress,
  ...
}
```

This lookup table would be reused a bunch of times in the next 80 lines of code or so.